ftw/book/upgrades/20191216174215_update_resource_bundle/upgrade.py

```diff
@@ -6,4 +7,8 @@ class UpdateResourceBundle(UpgradeStep):
     """
 
     def __call__(self):
-        self.install_upgrade_profile()
+        if IS_PLONE_5:
+            self.install_upgrade_profile()
+        else:
```

Comment:
You don't need the else part

Reply:
I kind of like to have it there to emphasise on the fact that in plone4 this should not be done and explicitly comment that. But I agree that it is pretty useless in means of it not doing anything.

Reply:
Why not just put the comment above the "if" then XD Something like this.
```
# The registry entries only have to be installed on plone 5
if IS_PLONE_5:
   ...
```

Reply:
That a good approach, thanks for the advise. I changed it in 1380d0c